topydo/lib/Config.py

```diff
@@ -266,6 +267,22 @@ def auto_creation_date(self):
         except ValueError:
             return self.defaults['auto_creation_date'] == '1'
 
+    def aliases(self):
+        """
+        Returns dict with aliases names as keys and pairs of actual
+        subcommand and alias args as values.
+        """
+        aliases = list(set(self.cp.items('aliases')) - set(self.cp.defaults().items()))
```

Comment:
* What is the purpose of this line exactly?
* Please add a comment with a clarification.
* Does it have to be a `list` ? Why not leave it as a set and iterate over that below? I don't think the order matters anymore after turning the configuration items into sets.